### DISCUSSION TITLE
Don't understand the error message

### DISCUSSION THREAD

**Discussion by mixiancmx** [PaddlePaddle/PaddleSpatial#268] [BaiduKDDCup2022] [answered]
![image](https://user-images.githubusercontent.com/83197963/179165406-e3fbc089-ede4-4faa-94bd-a4a84648c8a3.png)
I get this error message, but I don't understand. My code is based on Pytorch.

**Comment by shaido987** [comment, marked as answer]
Related questions that could help: https://github.com/PaddlePaddle/PaddleSpatial/discussions/135 and https://github.com/PaddlePaddle/PaddleSpatial/discussions/169.

Specifically, there is a backend code inspector that determines which framework the code uses. The fix seems to be to delete all unneeded files/scripts/imports in the submission, see https://github.com/PaddlePaddle/PaddleSpatial/discussions/169#discussioncomment-2961928.

(note that I'm just a participant and not one of the organizers so this is not an official response.)

**Reply by ramber1836** [reply]
Many thanks!

**Reply by shaido987** [reply]
No problems~ I guess you are all quite busy these days.

**Reply by ramber1836** [reply]
^_^

**Reply by mixiancmx** [reply]
Thanks!
